src/app/config/projectProperties.json

```diff
@@ -1,70 +1,49 @@
 {
     "projectTypes": {
-        "angular": {
-            "displayname": "Office Add-in project using Angular framework",
+        "basic-taskpane": {
+            "displayname": "Office Add-in Taskpane project",
```

Comment:
we should get content team input about the display names...

Reply:
In the meantime, please make Taskpane two words in the display names, e.g. "Task Pane". This is how it appears in the docs.